cli/get.go

```diff
@@ -166,7 +211,8 @@ A role must be specified when using this command through the --role flag. You ma
 			ttl = config.TTL
 		}
 
-		if cloudType == cloudAws {
+		switch cloudType {
```

Comment:
We can probably fold this `switch` statement into the previous one if we're smart